.editorconfig

```diff
@@ -80,3 +80,207 @@ dotnet_naming_rule.private_const_fields_ignore_camel_case_leading_underscore.sev
+dotnet_diagnostic.SA1017.severity = none			# Closing attribute brackets should be spaced correctly.
+dotnet_diagnostic.SA1018.severity = none			# Nullable type symbols should be spaced correctly.
+dotnet_diagnostic.SA1019.severity = none			# Member access symbols should be spaced correctly.
+dotnet_diagnostic.SA1020.severity = none			# Increment decrement symbols should be spaced correctly.
+dotnet_diagnostic.SA1021.severity = none			# Negative signs should be spaced correctly.
+dotnet_diagnostic.SA1022.severity = none			# Positive signs should be spaced correctly.
+dotnet_diagnostic.SA1023.severity = none			# Dereference and access of symbols should be spaced correctly.
+dotnet_diagnostic.SA1024.severity = none			# Colons should be spaced correctly.
+dotnet_diagnostic.SA1025.severity = none			# Code should not contain multiple whitespace in a row
+dotnet_diagnostic.SA1026.severity = none			# The keyword 'new' should not be followed by a space or a blank line.
+dotnet_diagnostic.SA1027.severity = none			# Use tabs correctly
+dotnet_diagnostic.SA1028.severity = none			# Code should not contain trailing whitespace.
+
+# StyleCop: Readability Rules
+dotnet_diagnostic.SA1100.severity = none			# Do not prefix calls with base unless local implementation exists
+dotnet_diagnostic.SA1101.severity = none			# Prefix local calls with this.
+dotnet_diagnostic.SA1102.severity = none			# Query clause should follow previous clause.
+dotnet_diagnostic.SA1103.severity = none			# Query clauses should be on separate lines or all on one line.
+dotnet_diagnostic.SA1104.severity = none			# Query clause should begin on new line when previous clause spans multiple lines.
+dotnet_diagnostic.SA1105.severity = none			# Query clauses spanning multiple lines should begin on own line.
+dotnet_diagnostic.SA1106.severity = none			# Code should not contain empty statements.
+dotnet_diagnostic.SA1107.severity = none			# Code should not contain multiple statements on one line.
+dotnet_diagnostic.SA1108.severity = none			# Block statements should not contain embedded comments.
+dotnet_diagnostic.SA1109.severity = none			# Block statements should not contain embedded regions.
+dotnet_diagnostic.SA1110.severity = none			# Opening parenthesis or bracket should be on declaration line.
+dotnet_diagnostic.SA1111.severity = none			# Closing parenthesis should be on line of last parameter.
+dotnet_diagnostic.SA1112.severity = none			# Closing parenthesis should be on line of opening parenthesis.
+dotnet_diagnostic.SA1113.severity = none			# Comma should be on the same line as previous parameter.
+dotnet_diagnostic.SA1114.severity = none			# Parameter list should follow declaration.
+dotnet_diagnostic.SA1115.severity = none			# Parameter should follow comma .
+dotnet_diagnostic.SA1116.severity = none			# Split parameters should start on line after declaration.
+dotnet_diagnostic.SA1117.severity = suggestion		# Parameters should be on same line or separate lines.
```

Comment:
Maybe not use tabs for the indentation.

Do we need rules for the .editorconfig itself ;)